DentOS_Framework/DentOsTestbed/configuration/testbed_config/basic_infra1/testbed.json

```diff
@@ -33,10 +33,10 @@
             "baudrate": 115200,
             "mediaMode":"mixed",
             "links" : [
-                ["10.36.118.199:1:9", "infra1:swp1", "copper"],
-                ["10.36.118.199:1:10", "infra1:swp2", "copper"],
-                ["10.36.118.199:1:11", "infra1:swp3", "copper"],
-                ["10.36.118.199:1:12", "infra1:swp4", "copper"]
+                ["10.36.118.199:2:1", "infra2:swp33", "copper"],
+                ["10.36.118.199:2:2", "infra2:swp34", "copper"],
+                ["10.36.118.199:2:3", "infra2:swp35", "copper"],
+                ["10.36.118.199:2:4", "infra2:swp36", "copper"]
```

Comment:
What about old links? As I understand, we do still have swp1-4 connected to Ixia card #1. Can we leave them here and just add new links?

Reply:
Also, you have infra2 here, although it is infra1 config